pkgs/development/tools/simavr/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, makeWrapper, avrgcclibc, libelf, which, git, pkgconfig, freeglut, mesa }:
+
+stdenv.mkDerivation rec {
+  name = "simavr-${version}";
+  version = "1.3";
+  enableParallelBuilding = true;
+
+  libPath = stdenv.lib.makeLibraryPath [
+    libelf
```

Comment:
Are you sure this is necessary? The build process should actually compile the correct path to libelf directly into the binary.